content/_releases/22/22.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Product Update - v22.22
+description: Details of the product update v22.22 from 2nd of June 2022.
+layout: rel
+version: "22.22"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-06-02
+redirect_from:
+  - /releases/22.22.html
+---
+
+## Announcements
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+### ISO 27001 certification
+
+> **We are ISO 27001 certified!** Check our
+> [dedicated page](/admin/iso27001-certification) for more details and download
+> the copy of our certificate.
+
+### Temporary Duplication of Execution Numbers
+
+As part of ongoing improvements we have refined our system architecture around
+executions to speed-up the load of Executions page. As part of these
+changes there is a temporary duplication of the number of executions recorded in
+your executions page. No data is lost, there is no influence on quota use, and
+these duplications will disappear as we end the transition phase in approximately
+30 days. If you have any questions please contact our support directly.
+
+### Attachment/object processing and storing
+
+We are changing policies for our retention services. As platform traffic grows
+the resource burden of retaining files becomes significant.
+
+*   The attachment and object (a.k.a. `lightweight objects`) retention period is reduced to 3 days (72 hours)
+*   The maximum attachment/object size is capped at 100 MB.
+
+{% endif %}
+
+## Features
+
+### Flow designer header
+
+To provide more vertical space during the flow design process we changed the flow
+designer header behaviour. Now when the left-hand-side menu is closed the action
+buttons and statuses merge up into the same line with flow title and description.
+
+{% include img.html max-width="100%" url="/assets/img/RN/2222/RN2222-flow-header-change.gif" title="Flow designer header" %}
+
+You can reveal the flow title or the description text if they are longer by
```

Comment:
When your flow title or description is larger than the available space the visible text is cropped. By hovering over these fields you can reveal the full text.